### ISSUE TITLE
Curator Affiliation - Affiliation display and selection

### ISSUE THREAD

**Issue by jimmyzhen** [ClinGen/clincoded#1508]
In the nav header, implement UI to display the selected affiliation of the current logged-in user if he/she is a member of an affiliation. For an user who belongs to multiple affiliations, implement UI to allow the user to select a different affiliation during the logged-in session.

These UI implementation shall be excluded from users who sign in as individuals.

**Comment by wrightmw** [comment]
@jimmyzhen When testing together, we found a bug. It's a very specific chain of events that generates the error.

Selina and I were both logged in as part of the same #100003 Affiliation (see https://r15-rc1.demo.clinicalgenome.org/curation-central/?gdm=8b5732d4-cfe7-4941-8940-04650a56a832&pmid=10000).
When both logged in everything worked fine, except some Probands became uneditable to us despite the fact that they were added by me as part of #100003:

<img width="189" alt="screen shot 2017-12-21 at 8 20 51 am" src="https://user-images.githubusercontent.com/15131169/34264477-e9c3c6c6-e627-11e7-816a-655c58f376d6.png">

We should both have been able to edit this because we were both #100003.

We found out that the only effected Probands were ones that were created as part of a Group and Family. If the proband was created separately then there were no issues.

A second surprising thing, is that only I could generate the error. If Selina created a Proband from a Group and Family, it worked just fine. We found out that it was because I was the creator of the PMID because when Selina added her own PMID, then she was able to recreate the problem:

![](https://files.slack.com/files-tmb/T0F6S42DA-F8K4LCQ7R-b14b51aa29/pasted_image_at_2017_12_21_08_12_am_160.png)

So, in short:
If a curator (associated with an Affiliation) adds a PMID and then they generate a Proband as part of a Group and Family, then that Proband becomes uneditable.

One more thing.... I just logged out of the Affiliation and logged in as myself and that Proband (that was created as part of #10003) is now now editable
